pom.xml

```diff
@@ -11,7 +11,7 @@
   <properties>
     <tycho.version>3.0.1</tycho.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <eclipse.target>2022-09</eclipse.target> <!-- TODO change to neon -->
+    <eclipse.target>2023-09</eclipse.target> <!-- TODO change to neon -->
```

Comment:
probably can remove that TODO as long as you're updating the line...

Reply:
removed todo